### PR TITLE
Provide device URL

### PR DESCRIPTION
**Pull request by chpego**
For the issue #603

### CODE REVIEW

File: custom_components/shelly/block.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ def device_info(self):
             'name': self._block.friendly_name(),
             'manufacturer': 'Allterco',
             'model': self._block.type_name(),
-            'sw_version': self._block.fw_version()
+            'sw_version': self._block.fw_version(),
+            'configuration_url': f'http://{self._block.ip_addr}'
```
Comment (chpego) [inline custom_components/shelly/block.py:107]: Maybe it's    :         'configuration_url': f"http://{self._block.ip_addr}" ?

File: custom_components/shelly/device.py [hunk 2]
```diff
@@ -164,7 +164,8 @@ def device_info(self):
             'name': self._dev.block.friendly_name(),
             'manufacturer': 'Allterco',
             'model': self._dev.type_name(),
-            'sw_version': self._dev.fw_version()
+            'sw_version': self._dev.fw_version(),
+            'configuration_url': f'http://{self._dev.ip_addr}'
```
Comment (chpego) [inline custom_components/shelly/device.py:168]: Maybe it's    :         'configuration_url': f"http://{self._block.ip_addr}" ?

File: custom_components/shelly/block.py (hunk 1 above)
Comment (hakana) [reply custom_components/shelly/block.py:107]: Did it not work for you? It was included in the release 0.3.1 beta 2. Or do you mean only for readability?

Comment (chpego) [reply custom_components/shelly/block.py:107]: Yes readibility

**Pull request merged**